modules/install/pages/sizing-general.adoc

```diff
@@ -220,7 +220,9 @@ Note that this amount is in addition to the RAM requirements for the operating s
 
 When sizing, you must account for raw CPU overhead when using a high number of buckets.
 
-* We recommend that you set aside 0.2 core/bucket for operational stability. This overhead does not account for any front-end workload which must be sized separately.
+* Your best practice is to allocate 0.4 cores per bucket on each node to maintain operational stability. 
```

Comment:
Do we really not use "we" any more for best practices? "Your best practice" seems awkward, and a little passive aggressive to me. But I guess needs must, if we're stomping down on "we" 🤷 

Moot, as this is a backport, but just curious.

Reply:
Vale's guidance is to avoid first person plural ("we"). Vale is good. Vale must be obeyed. 

I suppose "The best practice..." could also work. But that's also a bit passive and begs the question how this wisdom came about. I tend to avoid using the company's name ("Couchbase's test has determined the best practice to be...") when it collides with the product name because it suggests our product has gained sentience and is telling you what to do. 

(I don't think it's slated to do that until after version 10.0 or so.)

Reply:
I for one welcome our robotic/AI overlords